Dockerfile

```diff
@@ -6,17 +6,19 @@ WORKDIR ${PKG}
 RUN apt-get update && apt-get install -y make golang-glide git
 
 # resolve and install imports
-ADD glide.yaml glide.lock ${PKG}
+COPY glide.yaml glide.lock ${PKG}
 RUN glide install --strip-vendor
 
 # compile test binary
-ADD . ${PKG}
+COPY . ${PKG}
 
 RUN make check
 
 RUN make out/osde2e
 
 FROM gcr.io/distroless/base
+
+COPY artifacts artifacts
```

Comment:
We now have YAML artifacts we need copied into the container. Typically it's better to use COPY over ADD, so I updated the above lines as well. Functionally they're the same.